docker.go

```diff
@@ -5,7 +5,7 @@ import (
 	"encoding/json"
 	"errors"
 	"io/ioutil"
-	"path/filepath"
+	slashpath "path"
```

Comment:
Why the rename?

Reply:
Otherwise it steals a perfectly good variable name (`path`) from the program. I do this out of force of habit, but happy to remove the package name alias if you like.